plugins/Amplifier/Amplifier.cpp

```diff
@@ -28,95 +28,54 @@
 #include "embed.h"
 #include "plugin_export.h"
 
-extern "C"
-{
-
-Plugin::Descriptor PLUGIN_EXPORT amplifier_plugin_descriptor =
-{
-	STRINGIFY( PLUGIN_NAME ),
-	"Amplifier",
-	QT_TRANSLATE_NOOP( "PluginBrowser", "A native amplifier plugin" ),
-	"Vesa Kivimäki <contact/dot/diizy/at/nbl/dot/fi>",
-	0x0100,
-	Plugin::Effect,
-	new PluginPixmapLoader("logo"),
-	NULL,
-	NULL
-} ;
+extern "C" {
 
+Plugin::Descriptor PLUGIN_EXPORT amplifier_plugin_descriptor = {STRINGIFY(PLUGIN_NAME), "Amplifier",
+	QT_TRANSLATE_NOOP("PluginBrowser", "A native amplifier plugin"), "Vesa Kivimäki <contact/dot/diizy/at/nbl/dot/fi>",
+	0x0100, Plugin::Effect, new PluginPixmapLoader("logo"), NULL, NULL};
```

Comment:
This one looks a little bit weird.

Reply:
Thanks for your review. The only initializer thing I found which is not about CTOR initializers is `AllowAllArgumentsOnNextLine`, but changing it to `false` had no effect on this file. So this can not be fixed inside `.clang-format`.

However, we can add a comma after the last element (I just pushed a successfull fix for that file), this lets clang-format align them line by line. Should we (prior to the final clang-format run) add a comma for all the initializer lists that should have one initializer per line?

Opinions?

Reply:
If this looks right, we can do this for every other plugins.

Reply:
Created #6123 . Resolving this.